packages/suite/src/components/suite/CoinsGroup/CoinsList.tsx

```diff
@@ -62,11 +63,14 @@ const CoinsList = ({
                     unavailable && getUnavailabilityMessage(unavailable, features?.major_version);
                 const anyTooltip = lockedTooltip || unavailabilityTooltip || tooltip;
 
+                // When in bootloader mode we cannot check version of firmware so we do not know if coin is available.
+                // In order to achieve consistency between devices we do not use it when in bootloader mode.
+                const isUnavailable = !!unavailable && !isBootloaderMode;
                 // Coin is not available because:
                 // - connect reports this in device.unavailableCapabilities (not supported by fw, not supported by connect)
                 // - suite considers device 'locked'
                 // - suite does not support it which is defined in network.ts
-                const disabled = !!unavailable || locked || !supportedBySuite;
+                const disabled = isUnavailable || locked || !supportedBySuite;
 
                 return (
                     <Tooltip
```

Comment:
tooltip needs to be handled too (Trezor T, latest FW shows "Update required" for all coins when in bootloader mode)
<img width="308" alt="image" src="https://user-images.githubusercontent.com/3729633/155698296-917e001b-199b-4a10-b9f2-b7abbbfc73c7.png">

Reply:
ohh, good catch! I will handle that!

Reply:
@matejkriz it should be good now